plugins/cost-insights/src/hooks/useScroll.tsx

```diff
@@ -15,26 +15,22 @@
  */
```

Comment:
I refactored this hook slightly. 

In order to open the collapsed section when an alert status button is clicked, the `AlertInsights` component needed access to the scroll state. 

Since every `ScrollAnchor` implements the same `top` and `behavior` props (offset 20px from the top, smooth scrolling), it made sense to make those values defaults and just require the ScrollAnchor to define an id.